### ISSUE TITLE
Problem Using With bento/centos-6.9

### ISSUE THREAD

**Issue by warren5236** [gael-ian/vagrant-bindfs#83]
Hello,
I'm using bento/centos-6.9 for a vagrant box and when I attempt to vagrant up I receive the following error message:

```
==> centos: Checking for bindfs in VM...
    centos: Package manager detected: yum
==> centos: Fuse seems to not be installed on the virtual machine, installing now
    centos: Fuse kernel module is installed
==> centos: Fuse kernel module seems to be not loaded on the virtual machine, loading now
    centos: Fuse kernel module is loaded
==> centos: Bindfs seems to not be installed on the virtual machine, installing now
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

yum -y install bindfs

Stdout from the command:

Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * base: mirrors.gigenet.com
 * extras: mirrors.umflint.edu
 * updates: linux.mirrors.es.net
No package bindfs available.


Stderr from the command:

Error: Nothing to do
```

Any idea on how to solve this?

```
vagrant -v
Vagrant 2.1.5
vagrant plugin list
vagrant-bindfs (1.1.0, global)
vagrant-hostsupdater (1.1.1.160, global)
```

**Comment by gael-ian** [comment]
Hello,

As a workaround, you can [force bindfs installation from source](https://github.com/gael-ian/vagrant-bindfs#bindfs_version-and-install_bindfs_from_source) . I'll try to find some time to take a look at what happened to the bindfs RPM package in CentOS 6 (and 7) as soon as I can.

**Comment by gael-ian** [comment]
The bindfs RPM package is available in the EPEL repository, that may not have been activated in your box.

How to active the EPEL repository on CentOS:

* [CentOS Wiki](https://wiki.centos.org/AdditionalResources/Repositories#head-6d2fe2c161f17b9ca34f13298c09920e7d0cbbef)
* [Fedora Wiki](https://fedoraproject.org/wiki/EPEL#Quickstart)

First problem: both CentOS and Fedora are considered the same in `vagrant-bindfs` (Red Hat boxes with yum as package manager) but Fedora boxes don't need this.
Second problem: it may not be a good idea to active it on every CentOS boxes while installing bindfs.

The simple fact you runned into this problem prove [this test to find if bindfs is available as a RPM package](https://github.com/gael-ian/vagrant-bindfs/blob/master/lib/vagrant-bindfs/vagrant/capabilities/redhat/bindfs.rb#L9) is wrong.

**Comment by brunowego** [comment]
@gael-ian alpine (apk) support 👍

**Comment by gael-ian** [comment]
Hello @brunowego 

I suppose your short message means you'd like to see vagrant-bindfs to support Alpine Linux guest machines. In this case, please take the time to open a dedicated issue and, if you can, to find a test box available on Vagrant Cloud and well maintained (as asked in [vagrant-bindfs' Vagrantfile](https://github.com/gael-ian/vagrant-bindfs/blob/master/Vagrantfile#L21). Sadly, there is no official Alpine Linux box). A link to the Alpine package manager documentation can greatly help too, as I'm not user of this OS and know nothing about its arcanes.

Thanks

**Comment by gael-ian** [comment]
HI @warren5236,

I'm sorry that I did not took the time to fix this issue earlier but the new 1.1.1 release is now available. The problem was in the bindfs package detection _via_ yum: in CentOS 7 it returns "No matches found" when it doesn't found an appropriate package, in CentOS 6 it returns "No Matches found"...

I switched to another method that detect when the package is available instead of when it's not and now the problem is fixed.

Thank you for your patience

**Comment by warren5236** [comment]
Thank you for providing such an awesome plugin!
